roadmap/implementers-guide/src/node/availability/bitfield-distribution.md

```diff
@@ -6,16 +6,24 @@ Validators vote on the availability of a backed candidate by issuing signed bitf
 
 `ProtocolId`: `b"bitd"`
 
-Input: [`BitfieldDistributionMessage`](../../types/overseer-protocol.md#bitfield-distribution-message)
+Input:
+[`BitfieldDistributionMessage`](../../types/overseer-protocol.md#bitfield-distribution-message) which are gossiped to all peers, no matter if validator or not.
+
 Output:
 
-- `NetworkBridge::RegisterEventProducer(ProtocolId)`
+- `NetworkBridge::RegisterEventProducer(ProtocolId)` in order to register ourselfs as an event provide for the protocol.
 - `NetworkBridge::SendMessage([PeerId], ProtocolId, Bytes)`
-- `NetworkBridge::ReportPeer(PeerId, cost_or_benefit)`
-- `BlockAuthorshipProvisioning::Bitfield(relay_parent, SignedAvailabilityBitfield)`
+- `NetworkBridge::ReportPeer(PeerId, cost_or_benefit)` improve or penalize the reputiation of peer based on the message we received relative to our state.
+- `BitfieldDistributionMessage::DistributeBitfield(relay_parent, SignedAvailabilityBitfield)` gossip a verified incoming bitfield on to interested subsystems within this validator node.
```

Comment:
Is this saying that the `BitfieldDistribution` subsystem will output its own message?

Reply:
An initial misconception.